client-app/src/css/DonorForm.css

```diff
@@ -51,6 +51,111 @@
     color: #333;
 }
 
+Image upload and preview styling
+.donor-form .image-upload-container {
+    background-color: #0b0c10;
+    padding: 15px;
+    border-radius: 8px;
+    border: 1px solid #66fcf1;
+    margin-top: 15px;
+    position: relative;
+}
+
+.donor-form .image-preview {
+    display: flex;
+    flex-wrap: wrap;
+    gap: 15px;
+    margin-top: 15px;
+}
+
+.donor-form .image-preview img {
+    max-width: 100px;
+    max-height: 100px;
+    border-radius: 8px;
+    object-fit: cover;
+    transition: transform 0.3s ease, box-shadow 0.3s ease;
+}
+
+.donor-form .image-preview img:hover {
+    transform: scale(1.05);
+    box-shadow: 0 6px 12px rgba(0, 0, 0, 0.3);
+}
+
+/* Delete button for the images */
+.donor-form .delete-image-btn {
```

Comment:
@miabu-pashh Something went wrong with Donor Form, styles might be overridden. Please revert the input boxes to be in consistent design with white background and black text color
Form in your branch:
![image](https://github.com/user-attachments/assets/b248b5b7-126b-4437-bbd2-006c4cc7d579)
Same form in Main branch:
![image](https://github.com/user-attachments/assets/3d94c0ce-594e-48e1-bf82-4f80b1a060a9)